developer/src/common/web/utils/src/compiler-interfaces.ts

```diff
@@ -5,7 +5,13 @@ import { CompilerCallbacks } from "./compiler-callbacks.js";
  */
 export interface CompilerEvent {
   filename?: string;
+  /** line where a message applies */
   line?: number;
+  /** 
+   * column where a message applies.
+   * where line is unset, column is a file offset
```

Comment:
I think I'd prefer to have a separate property. They have to be handled differently anyway by other services

Reply:
OK, sounds good. I will leave column in even if we don't expose it at present.